conf/messages

```diff
@@ -1331,19 +1331,19 @@ property-details.multiple-disposals.guidance.capacitor.p3=Contract exchange usua
 property-details.multiple-disposals.guidance.p4=You’ll also need to upload a document later providing details of the other properties you’re including in this return.
 
 enterPostcode.returns.singleDisposal.title=Find the address of the UK property you disposed of
-enterPostcode.returns.agent.singleDisposal.title=Find the address of the UK property your client disposed of
+enterPostcode.returns.agent.singleDisposal.title=Find the address of the UK property the person disposed of
```

Comment:
does this content change apply to all agents or just agents of period of admin people?

Reply:
As per story description " The user enters this user journey when they answer 'Who are you completing this return for?' with 'The estate of a person who died before disposing of a property."

I think it applies to "Agent of an Estate" only.

Reply:
you need a separate user key to cover this then. I used `.personalRepInPeriodOfAdmin.agent` for mine when I had something similar - the pattern match for the user key will have to then have a case to return that value.

Reply:
if it helps this is what I ended up doing: https://github.com/hmrc/cgt-property-disposals-frontend/blob/master/app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/disposaldetails/disposal_fees.scala.html#L63